subsys/bluetooth/controller/ll_sw/lll_adv.h

```diff
@@ -129,10 +127,11 @@ struct lll_adv_aux {
 	 */
 	uint16_t data_chan_counter;
 
-	/* Temporary stored use by primary channel PDU event to fill the
+	/* Store used by primary channel PDU event to fill the
 	 * auxiliary offset to this auxiliary PDU event.
 	 */
 	uint32_t ticks_offset;
+	uint32_t us_offset;
```

Comment:
These members names does not say anything about reference point for these offsets. Hence one is forced to get to struct and luckily read the comment above. I suggest to change these names.
```suggestion
	uint32_t prim_adv_ticks_offset;
	uint32_t prim_adv_us_offset;
```